src/devices/ant_antplus.c

```diff
@@ -0,0 +1,198 @@
+/** @file
+    ANT and ANT+ decoder.
+
+    Copyright (C) 2022 Roberto Cazzaro <https://github.com/robcazzaro>
+
+    This program is free software; you can redistribute it and/or modify
+    it under the terms of the GNU General Public License as published by
+    the Free Software Foundation; either version 2 of the License, or
+    (at your option) any later version.
+*/
+/**
+ANT and ANT+ decoder.
+
+ANT and ANT+ comunication standards are defined by a division of Garmin
+https://www.thisisant.com/ and used widely for low power devices.
+The ANT radio transmits for less than 150 µs per message, allowing a single
+channel to be divided into hundreds of time slots and avoiding collisions.
+ANT and ANT+ devices use a modified Shockburst protocol, in the 2.4GHz ISM band,
+with 160kHz deviation and 1Mbps data rates, GFSK encoded. The low level
+layer is not documented anywhere. ANT chips use an 8 byte key to generate a 2 byte
+network ID using an unspecified algorithm. Valid keys are only assigned by Garmin and
+require specific licensing terms (and in some cases a payment)
+ANT+ uses the basic ANT message structure but is a managed network with a specific
+network key and well defined device types, each sending "data pages" of 8 bytes
+with specific data for the device type. Most ANT+ devices are sports focused like
+heart rate monitors, bicycle sensors, or environmental sensors.
+
+Please note that unlike most devices in the rtl_433 repository, ANT+ devices
+operate in the ISM band between 2.4GHz and 2.5GHz. Decoding these signals
+requires an SDR capable of operating above 2.4GHz (e.g. PlutoSDR) or the
+use of a downconverter for rtl_sdr. Finally, since the protocol encodes at
```

Comment:
It would be interesting if this could be used for this case. 

https://www.rtl-sdr.com/tuning-an-r820t2-rtl-sdr-up-to-6-ghz-via-a-harmonic-mixing-driver-hack/

A proper filter seems expensive though.

Reply:
Before breaking down and buying the PlutoSDR, https://www.kd0cq.com/2016/04/part-ii-the-mod-receive-up-to-4-5ghz-on-your-rtlsdr-for-5-00/ and(http://blog.cyberexplorer.me/2014/01/sniffing-and-decoding-nrf24l01-and.html. I wrote a short note about it here https://github.com/robcazzaro/nRF24L01-SDR-testing

Both solutions worked, but with a lot of noise. And in any case, the rtl-sdr had too low a sampling rate to be useful for ANT and ANT+. I even looked into a mixer, but then more knowledgeable people said what you just said: need an expensive filter. A PlutoSDR looked like a much more palatable alternative :)